index.html

```diff
@@ -3210,17 +3210,10 @@ <h4>Semantic Interoperability</h4>
           </p>
 
           <p>
-The base JSON-LD Context file for this specification also includes an extra
-feature, using the <a data-cite="JSON-LD11/#default-vocabulary">`@vocab`</a>
-keyword, which ensures that any undefined term in a [=verifiable credential=] or
-a [=verifiable presentation=] is automatically mapped to a URL prefixed with
-`https://www.w3.org/ns/credentials/issuer-dependent#`. This is to allow early
-experimentation with terms during the development phase, without requiring a
-formal definition in every cycle of that experimentation. Note that developers
-SHOULD NOT use this feature in production; this could lead to name clashes,
-yielding semantic ambiguities with other applications. Instead, they SHOULD
-define all the terms, as described earlier in this section, to achieve proper
-interoperability.
+Developers SHOULD NOT use the <a data-cite="JSON-LD11/#default-vocabulary">`@vocab`</a> feature in production as it can lead to
+name clashes, yielding semantic ambiguities with other applications. Instead,
+they SHOULD define all the terms, as described earlier in this section, to
+achieve proper interoperability.
```

Comment:
```suggestion
Developers SHOULD NOT use the <a data-cite="JSON-LD11/#default-vocabulary">`@vocab`</a> feature in production as it can lead to
JSON term clashes, resulting in semantic ambiguities with other applications. Instead,
to achieve proper interoperability, developers SHOULD define all terms used by their
applications, as described earlier in this section.
```

Reply:
I updated your suggestion with a ref to the relevant section.